packages/http/libraries/core/src/adapter/InversifyHttpAdapter.ts

```diff
@@ -0,0 +1,209 @@
+
+        return {
+          handler: async (
+            req: TRequest,
+            res: TResponse,
+            next: TNextFunction,
+          ): Promise<unknown> => {
+            try {
+              const handlerParams: unknown[] = await Promise.all(
+                this.#buildHandlerParams(
+                  parameterMetadata?.[
+                    controllerMethodMetadata.methodKey as string
+                  ] ?? [],
+                  req,
+                  res,
+                  next,
+                ),
+              );
+
+              const value: unknown = await (
+                controller[controllerMethodMetadata.methodKey] as (
+                  ...args: unknown[]
+                ) => unknown
+              )(...handlerParams);
+
+              return this._reply(req, res, value);
+            } catch (error) {
+              next(error);
+              return undefined;
+            }
+          },
+          methodKey: controllerMethodMetadata.methodKey,
+          path: controllerMethodMetadata.path,
+          requestMethodType: controllerMethodMetadata.requestMethodType,
+        };
+      },
+    );
+  }
+
+  #buildHandlerParams(
+    controllerMethodParameterMetadataList: ControllerMethodParameterMetadata[],
+    request: TRequest,
+    response: TResponse,
+    next: TNextFunction,
+  ): Promise<unknown>[] {
+    return controllerMethodParameterMetadataList.map(
+      async (
+        controllerMethodParameterMetadata: ControllerMethodParameterMetadata,
+      ) => {
+        switch (controllerMethodParameterMetadata.parameterType) {
+          case RequestMethodParameterType.BODY:
+            return this._getBody(
+              request,
+              controllerMethodParameterMetadata.parameterName,
+            );
+          case RequestMethodParameterType.REQUEST: {
+            return request;
+          }
+          case RequestMethodParameterType.RESPONSE: {
+            return response;
+          }
+          case RequestMethodParameterType.PARAMS: {
+            return this._getParams(
+              request,
+              controllerMethodParameterMetadata.parameterName,
+            );
+          }
+          case RequestMethodParameterType.QUERY: {
+            return this._getQuery(
+              request,
+              controllerMethodParameterMetadata.parameterName,
+            );
+          }
+          case RequestMethodParameterType.HEADERS: {
+            return this._getHeaders(
+              request,
+              controllerMethodParameterMetadata.parameterName,
+            );
+          }
+          case RequestMethodParameterType.COOKIES: {
+            return this._getCookies(
+              request,
+              controllerMethodParameterMetadata.parameterName,
+            );
+          }
+          case RequestMethodParameterType.NEXT: {
+            return next;
+          }
+          case RequestMethodParameterType.PRINCIPAL: {
```

Comment:
I'm curious about this :)

Reply:
This functionality returns the user from the request in the current library. We currently do not have an authentication service, which is why it is disabled.

Anyways, if we enable authentication services, do we want to keep this functionality with the same name and usage? I'm not sure.